hnn_core/viz.py

```diff
@@ -1150,8 +1166,15 @@ def plot_laminar_csd(times, data, contact_labels, ax=None, colorbar=True,
     import matplotlib.pyplot as plt
     if ax is None:
         _, ax = plt.subplots(1, 1, constrained_layout=True)
-
-    im = ax.pcolormesh(times, contact_labels, np.array(data),
+    plot_data = []
+    plot_times = []
+    for time_data in data:
+        print("len(times)", len(times))
+        print("len(time_data)", len(time_data))
```

Comment:
debug artifacts?

Reply:
Sorry, forgot to removed them while pushing. I've done it now